charts/unifi-poller/templates/configmap.yaml

```
@@ -1,9 +1,11 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "unifi-poller.fullname" . }}
+  name: {{ template "common.names.fullname" . }}-settings
   labels:
-{{ include "unifi-poller.labels" . | indent 4 }}
+    {{- include "common.labels" . | nindent 4 }}
 data:
-  up.yaml: {{ .Values.config | toYaml | quote }}
-
+  unifi-poller.yaml: |
+{{- with .Values.config }}
+    {{- toYaml . | nindent 4 }}
+{{- end }}
```

```suggestion
  {{- with .Values.config }}
    {{- toYaml . | nindent 4 }}
  {{- end }}
```
Just nitpicking here...